src/ai/susi/server/api/cms/ListSkillService.java

```diff
@@ -240,26 +253,29 @@ public ServiceResponse serviceImpl(Query call, HttpServletResponse response, Aut
                     JSONObject skillMetadata = SusiSkill.getSkillMetadata(model_name, group_name, language_name, skill_name, duration);
                     totalSkills++;
 
-                    if(reviewed.equals("true")) {
-                        if(SusiSkill.getSkillReviewStatus(model_name, group_name, language_name, skill_name)) {
-                            jsonArray.put(skillMetadata);
-                            skillObject.put(skill_name, skillMetadata);
-                            reviewedSkills++;
-                        }
-                        else {
-                            nonReviewedSkills++;
+                    if(reviewed.equals("true") || staff_picks.equals("true")) {
```

Comment:
This very large piece of code is identical to the one above. Please remove the code from both places, make a method and call the method instead. 
Copy-Paste Code is very dangerous! It will most certainly cause problems once bugfixes must be done.